src/app/application.h

```diff
@@ -112,6 +112,7 @@ class Application final : public BaseApplication
 
 private slots:
     void processMessage(const QString &message);
+    void torrentAdded(BitTorrent::Torrent *const torrent);
```

Comment:
```suggestion
    void torrentAdded(const BitTorrent::Torrent *torrent) const;
```
I suppose this event shouldn't be modifying the class members.

Reply:
You are absolutely right, but then again, all events could be considered for this very change.